.coveragerc

```diff
@@ -0,0 +1,6 @@
+[run]
+source = kuma
+branch = 1
```

Comment:
I don't know what these do. And I can't find any docs on Travis that explains them.